### PR TITLE
Cleaned up test.sh

### PR DESCRIPTION
**Pull request by marcusnaslund**
I like how we had three styles for if-statements. Now just one.

Peer review @fredrikbryntesson

### CODE REVIEW

**Comment by sebastianbaginski** [comment]
Why this style ? I think keeping `if ; then` on a single line is more readable. Kind of like
```C
if () {
}
```
vs
```C
if ()
{
}
```

**Comment by marcusnaslund** [comment]
Sure, but there were a lot more of this style than the other. I can change them to one-line if that's "better".

**Comment by sebastianbaginski** [comment]
Looks like it's time for a `bash` chapter in the style guide ;)

**Comment by marcusnaslund** [comment]
Actually, in that case, we should do `for` and `while` loops on the same line as well. Should we? Can we?

**Comment by sebastianbaginski** [comment]
I think we can, but it sounds like something to discuss with @thomasfanell and @fredrikbryntesson ?

**Comment by marcusnaslund** [comment]
Agreed. Postponed until monday. Have a nice weekend. :)

**Comment by marcusnaslund** [comment]
It's decided. Peer review @sebastianbaginski ?

File: test.sh [hunk 1]
```diff
@@ -40,15 +42,12 @@ do
...
 	./Tests
 	exitcode=$?
 fi
-if [[ !( $exitcode == 0 ) ]]
-then
-	exit 1
-else
-	exit 0
-fi
+exit $exitcode
```
Comment (thomasfanell) [inline test.sh]: You might want to change `test/Tests.ooc` to reflect this change then, since it wants either a 0 or a 1.

Comment (marcusnaslund) [inline test.sh]: How do you mean? `Tests.ooc` does not read output from `test.sh`, it's the other way around.

Comment (thomasfanell) [inline test.sh]: Well that was embarrassing. Pardon me while I go inject 10 cups of coffee directly into my brain.

Comment (marcusnaslund) [inline test.sh]: Everything is forgiven on Mondays. :)

**Pull request merged**